pages/common/protoc.md

```diff
@@ -0,0 +1,19 @@
+# protoc
+
+> Parse Google Protobuf `.proto` files and generate output in the specified language.
+
+- Generate Python code from a `.proto` file:
+
+`protoc --python_out={{output_directory}} {{input_file}}`
```

Comment:
Change `output_directory` to `path/to/output_directory` everywhere.

Also change `input_file` to `input_file.proto` for clarity.

Reply:
fixed